.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+### Description
+
+_Please explain the changes you made here._
+_Explain the **motivation** for making this change. What existing problem does the pull request solve?_
+_Mention any issues fixed, addressed or otherwise related to this pull request, including issue numbers or hard links for issues in other repos._
+
+----
+
+### Checklist (never delete this)
+
+Never delete this, it is our record that procedure was followed. If you find that for whatever reason one of the checklist points doesn't apply to your PR, you can leave it unchecked but please add an explanation below. 
+
+#### Content
+- [ ] Added or modified tests to cover changes and any new functionality
+- [ ] All tests passing on Travis 
+
+#### Form
+- [ ] Rebased, squashed and reworded to produce a single commit (exemptions may be made -- sparingly) that is _uncluttered by excess lines_ (lines that just say "addressed review", "fixed" etc left over from squashing)
+- [ ] Both the PR and the final rebased commit have a **concise** yet **descriptive** title
+- [ ] Edited the README / documentation accordingly
+
+#### Review
+- [ ] Suggest a reviewer if you can
+- [ ] Final (thumbsup) from the reviewer(s)
```

Comment:
Or the reviewer, I don't care. But I do think a human should check it. 

Checklists are redundant by design. Point is to summarize the information. Let's say the reviewer didn't use a thumbsup symbol, or further discussion occurred afterward. If you go back to the issue to see if it was properly okayed, you have to scan through the whole discussion. With a checklist, you just have to look at one line. 

If everyone hates this I won't push the issue. But it's how I would run the world if I was a dictator (technically I am, just not in this repo).